src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Don't reset your styles with * selector, move it to reset.css and reset only for specific tags